cpp/models/ode_seir/model.h

```diff
@@ -63,6 +70,55 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
         dydt[(size_t)InfectionState::Recovered] =
             (1.0 / params.get<TimeInfected>()) * y[(size_t)InfectionState::Infected];
     }
+
+    /**
+    *@brief After the simulation is finished and we get a resulting TimeSeries, this function uses the data to compute the reproduction number
+    *at an arbitrary time
+    *@param timeindex The time at which we want to compute the reproduction number
+    *@param y The TimeSeries. We actually only use the number of Susceptibles at a given time
+    *@returns The reproduction number in the seir model
+    *@see The same functions in the model.h files of the secir and secirvvs models
+    */
+
+    IOResult<double> get_reproduction_number(Eigen::Index timeindex, const mio::TimeSeries<ScalarType>& y) noexcept
```

Comment:
```suggestion
    IOResult<double> get_reproduction_number(double t_idx, const mio::TimeSeries<ScalarType>& y) noexcept
```
Shorter name for param. Why do you use Eigen::Index here? Double would make your life easier here :)